doc/sphinx/source/imas/index.rst

```diff
@@ -0,0 +1,67 @@
+| wall/description_2d/vessel/unit/annular/resistivity  | Wall resistivity (scalar)                                     |
++------------------------------------------------------+---------------------------------------------------------------+
+| equilibrium/time_slice/profiles_2d/psi               | Poloidal flux (from 2D equilibrium, vector)                   |
++------------------------------------------------------+---------------------------------------------------------------+
+
+The wall radius is a user specified quantity which must be specified when calling the IDS read in function. The wall radius and wall resistivity are used to calculate the inverse wall time for the boundary condition of the electric field equation in case a DREAM settings object is initialized.
+The radial coordinate used to initialize the DREAM settings object is the mid-plane outer radius. This is acquired by taking the poloidal flux coordinates from the 2D equilibrium IDS on the outer mid-plane. The core profiles radial grid as a function of core profiles poloidal flux is then interpolated onto the outer mid-plane flux to make sure the correct radial grid is used. The plasma profiles located in the core profiles IDS are then interpolated onto the new radial grid in DREAM.
+
+.. code-block:: python
+
+   def readInIDSSlice(shot, run, tokamak, user=os.getlogin(), time=-999, log=False, setUpDream=True, wall_radius=-999):
+   
+The ``readInIDSSlice()`` function takes a single time slice from the IDS database and loads in the data as an initial condition for a DREAM simulation. The shot and run numbers of the input IDS has to be specified as well as the user from whose IMAS database the shot file will used. The time can also be specified by the user, otherwise the earliest time point for which all the required IDS fields are filled will be loaded. Logging of the IDS loading can be requested. This creates a log file, informing on which of the required fields are loaded successfully.
+
+The ``setUpDREAM()`` option can be used to initialize a DREAM settings object as an output from the function. The object initializes a DREAM run with the initial conditions for the physical quantities set from the IDS and the temperature and electric field calculated self-consistently. A few other settings might be required to be set independently from the IDS read in function. A simple example on how to use IDS's with DREAM can be found in the examples directory in the DREAM repository (``examples/imas``). This is a Python script which creates a ``dream_settings.h5`` file which can be used to run DREAM with the necessary input. The execution of the example file requires the ``imas`` Python module to be installed. The input generation from IDS's is conducted independently from running DREAM with the settings file, and can be run separately.
+
+The option not to create a DREAM settings object is currently not implemented, the loaded data could be saved in Python lists or dictionaries.
```

Comment:
Do you mean line 57? I am trying to say that currently it is not possible to run the interface without also generating a DREAMSettings object. It is probably not that useful as I assume most users would want to generate this anyway.
It also probably doesn't make sense to not create the settings object, as all the data is available in the input IDS if one knows where and how to get it. But it might be useful to add this if we think it useful.
Should I modify/remove this line?

Reply:
Based on one of your comments below, I have modified this line, as now the variables are saved in a dictionary.